Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -391,6 +391,12 @@ class SmartPage_Context
 
     var isAsynchronous = false;
     this.PageLoaded(isAsynchronous);
+
+    let data = {
+      ...history.state,
+      pageToken: document.getElementById('SmartPage_InstanceToken')!.getAttribute('value') ?? '',
```

Comment:
see SmartPage.ts, there's a helper API `this.GetFormElement(id).value` we should use.